app/src/main/res/values/untraslatable.xml

```diff
@@ -133,6 +135,10 @@
 		<item quantity="one">@string/TestResults_Summary_Websites_Hero_Reachable_Singular</item>
 		<item quantity="other">@string/TestResults_Summary_Websites_Hero_Reachable_Plural</item>
 	</plurals>
+	<plurals name="TestResults_Overview_Circumvention_RiseupVPN_Blocked" translatable="false">
```

Comment:
@cyBerta we still use the plurals but in this file. If you want to help improve our translation system I can open a ticket and we can discuss it there, apprecciated!